core-bundle/src/Migration/MigrationResult.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Migration;
+
+class MigrationResult
```

Comment:
should this data object be `final`?

Reply:
Sounds reasonable to me to mark value objects `final`, @contao/developers WDYT?

Reply:
Done in cfb6404161f2a504c4e23af55cca52034307492a